Makefile

```diff
@@ -105,7 +105,7 @@ generate-type-doc:  ## Generate typedoc.
 
 .PHONY: generate-api-types
 generate-api-types:  ## Generate api-types.
-	cd frontend && npm run generate-api-types
+	-cd frontend && npm run generate-api-types
```

Comment:
There is a workaround here, I m not sure how to solve this yet. When using the makefile in CI now, we perform generate-api-types for the storybook, but we dont have typeshare installed in that docker, so in CI they are moved over stages, but locally we want to just create the types fresh, maybe just remove type generation.

Reply:
Why not simply depend on the generated file? Then it should not matter if it was actually generated (local builds) or copied from other stages (CI).

Reply:
Totally forgot about that.
I didnt manage to do it as [pre-requisite](https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html), so I added this if.

I basically tried this:
```
.PHONY: generate-api-types
generate-api-types:  ./frontend/src/bindings/definitions.ts  ## Generate api-types.
	my-code
```

Reply:
Do we maybe have a misunderstanding about the term pre-requisite? With pre-requisite I meant text written by us in the documentation that a user must have install packages (or use our Docker file) before using our Makefiles.

Reply:
> Why not simply depend on the generated file?

With pre-requisite I mean the file dependance you mentioned.